src/org/jetbrains/plugins/scala/project/template/Downloader.scala

```diff
@@ -62,6 +70,14 @@ object Downloader {
       s"""set libraryDependencies := Seq("ch.epfl.lamp" % "dotty_2.11" % "$version" % "scala-tool")""",
       "updateClassifiers")
   }
+
+  private def sbtCommandsForHydra(platform: Platform, version: String) = Seq(
+    s"""set scalaVersion := "${version.split("_")(0)}"""",
+    s"""set credentials := Seq(Credentials("Artifactory Realm", "repo.triplequote.com", "${HydraCredentialsManager.getLogin}", "${HydraCredentialsManager.getPlainPassword}"))""",
+    s"""set resolvers := Seq(Resolver.url("Triplequote Plugins Ivy Releases", url("https://repo.triplequote.com/artifactory/ivy-releases/"))(Resolver.ivyStylePatterns), Resolver.url("Triplequote sbt-plugin-relseases", url("https://repo.triplequote.com/artifactory/sbt-plugins-release/"))(Resolver.ivyStylePatterns),  "Triplequote Plugins Releases" at "https://repo.triplequote.com/artifactory/libs-release-local/")""",
+    s"""set libraryDependencies := Seq("com.triplequote" % "hydra_${version.split("_")(0)}" % "${version.split("_")(1)}", ("com.triplequote" % "hydra-bridge_1_0" % "${version.split("_")(1)}").sources())""",
```

Comment:
Should we use the [`GroupId`](https://github.com/triplequote/intellij-scala/pull/5/files#diff-db6c8aac04fc8678946c25d33f46f03aR17) constant here?